src/api/wix/WixToolset.Data/Symbols/SymbolDefinitions.cs

```diff
@@ -322,6 +322,9 @@ public static IntermediateSymbolDefinition ByType(SymbolDefinitionType type)
                 case SymbolDefinitionType.FileSFPCatalog:
                     return SymbolDefinitions.FileSFPCatalog;
 
+                case SymbolDefinitionType.HarvestFiles:
```

Comment:
Should this change be in this PR? I'm fine if it tags along but it looks like it belongs with what I guess is your next fix. :)

Reply:
Yes. It's been missing for a while, unfortunately, and shows up because I used `Files` in the test.